fiboa_cli/datasets/be_wa.py

```diff
@@ -60,12 +62,13 @@
 }
 
 def migration(gdf):
-    gdf['crop_name'] = gdf['crop_name'].str.strip()  # .str.replace("  ", " ").str.replace("( ", "(")
-
     # Extract last digits from crop_code, example crop_code is:
     # http://geoservices.wallonie.be/inspire/atom/LU_LandUseClassification_LPIS.xml?code=#LandUseClass.lpis.cropCategory.6
     gdf['crop_code'] = gdf['crop_code'].str.extract(r'\.(\d+)$', expand=False)
+    gdf['crop_name'] = gdf['crop_name'].str.strip()  # .str.replace("  ", " ").str.replace("( ", "(")
     gdf["determination_datetime"] = "2022-01-01T00:00:00Z"
+    gdf["area"] = gdf["geometry"].area
```

Comment:
We don't do the automatic addition of area/perimeter usually. We likely add a generic parameter for this soon, so I'd propose to remove this here.

Reply:
Will do